src/aws-vault.ts

```diff
@@ -0,0 +1,168 @@
+    description: "Remove credentials from the secure keystore",
+    args: {
+      name: "profile",
+      generators: profilesGeneratro,
+    },
+    options: [
+      {
+        name: ["-f", "--force"],
+        description: "Force-remove the profile without a prompt",
+      },
+    ],
+  },
+  {
+    name: "list",
+    description: "List profiles, along with their credentials and sessions",
+    options: [
+      {
+        name: "--profiles",
+        description: "Show only the profile names",
+      },
+      {
+        name: "--sessions",
+        description: "Show only the session names",
+      },
+      {
+        name: "--credentials",
+        description: "Show only the profiles with stored credential",
+      },
+    ],
+  },
+  {
+    name: "rotate",
+    description: "Rotate credentials",
+    args: {
+      name: "profile",
+      generators: profilesGeneratro,
+    },
+    options: [
+      {
+        name: ["-n", "--no-session"],
+        description: "Use master credentials, no session or role used",
+      },
+    ],
+  },
+  {
+    name: "exec",
+    description: "Execute a command with AWS credentials",
+    args: [
+      {
+        name: "profile",
+        generators: profilesGeneratro,
+      },
+      {
+        name: "command",
+        isCommand: true,
+      },
+    ],
+    options: [
+      {
+        name: ["-d", "--duration"],
+        description:
+          "Duration of the temporary or assume-role session. Defaults to 1h",
+        args: {
+          name: "DURATION",
+        },
+      },
+      {
+        name: ["-n", "--no-session"],
+        description: "Use master credentials, no session or role used",
+      },
+      {
+        name: "--region",
+        description: "The AWS region",
+        args: {
+          name: "REGION",
+        },
+      },
+      {
+        name: ["-t", "--mfa-token"],
+        description: "The MFA token to use",
+        args: {
+          name: "MFA-TOKEN",
+        },
+      },
+    ],
+  },
+  {
+    name: "export",
+    description: "Export AWS credentials",
+    args: {
+      name: "profile",
+      generators: profilesGeneratro,
+    },
+  },
+  {
+    name: "clear",
+    description: "Clear temporary credentials from the secure keystore",
+    args: {
+      name: "profile",
+      generators: profilesGeneratro,
+    },
+  },
+  {
+    name: "login",
+    description: "Generate a login link for the AWS Console",
+    args: {
+      name: "profile",
+      generators: profilesGeneratro,
+    },
+  },
+];
+
+const helpSuggestions: Fig.Suggestion[] = subcommands.map(({ name }) => ({
+  name,
+  type: "subcommand",
+}));
+
+const completionSpec: Fig.Spec = {
+  name: "aws-vault",
+  description:
+    "A vault for securely storing and accessing AWS credentials in development environments",
+  subcommands: [
+    {
+      name: "help",
+      description: "Show help about the command",
+      args: {
+        name: "command",
+        suggestions: helpSuggestions,
```

Comment:
This is a nice sugar we provide :)
```suggestion
        template: "help",
```

Reply:
@fedeci 
Thanks! I fixed it, check it out.